src/main/drivers/timer_stm32f4xx.c

```diff
@@ -94,6 +94,10 @@ uint8_t timerClockDivisor(TIM_TypeDef *tim)
 
 uint32_t timerClock(TIM_TypeDef *tim)
 {
```

Comment:
This may be easier to extend/read - catch cpu-special cases first:
```
uint32_t timerClock(TIM_TypeDef *tim)
{
#if defined (STM32F411xE)
    return SystemCoreClock;
#elif defined (STM32F40_41xxx)
    if (tim == TIM8)
        return SystemCoreClock;
#endif
    if (tim == TIM1 || tim == TIM9 || tim == TIM10 || tim == TIM11) {
        return SystemCoreClock;
    }
    return SystemCoreClock / 2;
}
```